src/Dom/Document.php

```diff
@@ -635,22 +642,22 @@ public function normalizeDomStructure()
             }
 
             $this->body = $body;
```

Comment:
```suggestion
            $this->properties['body'] = $body;
```
?

Reply:
That will bypass the `__set()` method, but above this method is already doing `$this->head`. Since elsewhere in this method the magic getters/setters are being bypassed, probably good to do so here too.